src/style.css

```diff
@@ -1,3 +1,80 @@
+:root {
+  --link-color: #000;
+  --link-active-color: #00acdc;
+}
+
+html {
+  font-size: 10px;
+  font-family: Roboto, 'Arial', sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. The Arial font should be wrapped in quotes since it is a fallback font and not a generic family name. It's recommended to include a generic font family at the end as well.